controller_manager/src/controller_manager/controller_manager_interface.py

```diff
@@ -75,37 +75,28 @@ def unload_controller(name):
         return False
 
 def start_controller(name):
-    return start_stop_controllers([name], True)
+    return start_stop_controllers(start=[name])
```

Comment:
Was trying to follow the style of `controller_manager_interface.py`. Also `_controllers` doesn't give more information because the function is called `start_stop_controllers()`

Reply:
The reason I'd call `start_controllers([name])` is b/c if in the future sth is added there, it's better to have a single entry point. I think we'll always handle a single element the same way as a list of a single element.